### PR TITLE
[zh-cn] Add kube-scheduler-config.v1.md chinese version

### PR DESCRIPTION
**Pull request by zhangxyjlu**
fixed: #36476

Translate Chinese version for kube-scheduler-config.v1.md

Preview: https://deploy-preview-36490--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/config-api/kube-scheduler-config.v1/

### CODE REVIEW

File: content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md [hunk 1]
```diff
@@ -0,0 +1,1688 @@
...
+shortlist. The likely number of candidates enumerated for dry running
+preemption is given by the formula:
+numCandidates = max(numNodes * minCandidateNodesPercentage, minCandidateNodesAbsolute)
+We say &quot;likely&quot; because there are other factors such as PDB violations
+that play a role in the number of candidates shortlisted. Must be at least
+0 nodes. Defaults to 100 nodes if unspecified.
+   -->
+   <p><code>minCandidateNodesAbsolute</code> 字段设置 shortlist 中候选节点的绝对下限。
+   用于试运行抢占而列举的候选节点个数近似于通过下面的公式计算的：<br/>
+   候选节点数 = max(节点数 * minCandidateNodesPercentage, minCandidateNodesAbsolute)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md:72]:
```suggestion
   候选节点数 = max(节点数 * minCandidateNodesPercentage, minCandidateNodesAbsolute)</br>
```

File: content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md [hunk 2]
```diff
@@ -0,0 +1,1688 @@
...
+   之所以说是&quot;近似于&quot;是因为存在一些类似于 PDB 违例这种因素，
+   会影响到进入 shortlist中候选节点的个数。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md:74]:
```suggestion
   会影响到进入 shortlist 中候选节点的个数。
```

File: content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md [hunk 3]
```diff
@@ -0,0 +1,1688 @@
...
+<tr><td><code>bindVerb</code> <B><!--[Required]-->[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <!--
+   Verb for the bind call, empty if not supported. This verb is appended to the URLPrefix when issuing the bind call to extender.
+If this method is implemented by the extender, it is the extender's responsibility to bind the pod to apiserver. Only one extender
+can implement this function.
+   -->
+   <p>bind 调用所使用的动词，如果不支持过滤操作则为空。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md:577]:
```suggestion
   <p>bind 调用所使用的动词，如果不支持 bind 操作则为空。
```

File: content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md [hunk 4]
```diff
@@ -0,0 +1,1688 @@
...
+</td>
+</tr>
+<tr><td><code>prioritizeVerb</code> <B><!--[Required]-->[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <!--
+   Verb for the prioritize call, empty if not supported. This verb is appended to the URLPrefix when issuing the prioritize call to extender.
+   -->
+   <p>prioritize 调用所使用的动词，如果不支持过滤操作则为空。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md:552]:
```suggestion
   <p>prioritize 调用所使用的动词，如果不支持 prioritize 操作则为空。
```

File: content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md [hunk 5]
```diff
@@ -0,0 +1,1688 @@
...
+</td>
+</tr>
+<tr><td><code>preemptVerb</code> <B><!--[Required]-->[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <!--
+   Verb for the preempt call, empty if not supported. This verb is appended to the URLPrefix when issuing the preempt call to extender.
+   -->
+   <p>preempt 调用所使用的动词，如果不支持过滤操作则为空。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kube-scheduler-config.v1.md:541]:
```suggestion
   <p>preempt 调用所使用的动词，如果不支持 preempt 操作则为空。
```

**Review by tengqm — COMMENTED** [review]
非常好。

**Comment by windsonsea** [comment]
wow, great thanks.
/lgtm

**Pull request merged**